bin/load_db_scxa_dimred.sh

```diff
@@ -86,7 +86,7 @@ for dimred_type in tsne umap; do
 
   for f in $(ls $EXPERIMENT_DIMRED_PATH/$dimred_prefix*$DIMRED_SUFFIX); do
     paramval=$(echo $f | sed s+$EXPERIMENT_DIMRED_PATH/$dimred_prefix++ | sed s/$DIMRED_SUFFIX// )
-    tail -n +2 $f | awk -F'\t' -v EXP_ID="$EXP_ID" -v params="$dimred_param=$paramval" -v method="$dimred_type" 'BEGIN { OFS = ","; }
+    tail -n +2 $f | awk -F'\t' -v EXP_ID="$EXP_ID" -v params="[{\"$dimred_param\": $paramval}]" -v method="$dimred_type" 'BEGIN { OFS = ","; }
```

Comment:
If `$paramval` is text, wouldn't it require double quotes to be proper JSON?

Reply:
Right now, since dimred_param can only be perplexity or n_neighbours, then paramval can only be integer.

Reply:
> Given how critical this is, could you please add some testing to check that what you load as JSON comes back correctly on a query? Thanks.

Sure, will do. Though note that it's not possible to store invalid JSON in this field (as I found out when I malformed the UPDATES I used for testing).